specification/ai/ContentUnderstanding/scenarios.md

```diff
@@ -0,0 +1,247 @@
+# Champion Scenarios
+
+## Analyze a PDF using a prebuilt document analyzer (binary input)
+
+**Customer Value:** Extracts key content from local PDFs at scale to automate downstream workflows without manual data entry.
+
+**REST API Flow:**
+
+1. Start analysis with the prebuilt analyzer on binary content
+2. Poll operation status until completed
+3. Retrieve structured results (JSON)
+4. (Optional) Download any generated result files
+
+```http
+POST /analyzers/prebuilt-documentAnalyzer:analyze
+	Content-Type: application/pdf
+	Body: <binary PDF>
+
+GET /analyzers/prebuilt-documentAnalyzer/operations/{operationId}
+
+GET /analyzerResults/{operationId}
+
+GET /analyzerResults/{operationId}/files/{path}
+```
+
+---
+
+## Analyze a PDF and persist the raw JSON response
+
+**Customer Value:** Enables auditability and offline post-processing by saving the raw analysis JSON to storage.
+
+**REST API Flow:**
+
+1. Start analysis with the prebuilt analyzer on binary content
+2. Poll operation status until completed
+3. Retrieve structured results (JSON)
+4. Save JSON to storage (e.g., blob, filesystem)
+
+```http
+POST /analyzers/prebuilt-documentAnalyzer:analyze
+GET /analyzers/prebuilt-documentAnalyzer/operations/{operationId}
+GET /analyzerResults/{operationId}
+```
+
+---
+
+## Analyze a document from a URL
+
+**Customer Value:** Processes web-hosted or cloud-stored documents without downloading them locally.
+
+**REST API Flow:**
+
+1. Start analysis with a URL payload
+2. Poll operation status until completed
+3. Retrieve structured results (JSON)
+
+```http
+POST /analyzers/prebuilt-documentAnalyzer:analyze
+	Content-Type: application/json
+	{
+		"url": "https://.../document.pdf"
+	}
+
+GET /analyzers/prebuilt-documentAnalyzer/operations/{operationId}
+GET /analyzerResults/{operationId}
+```
+
+---
+
+## Extract invoice fields using the prebuilt-invoice analyzer (URL)
+
+**Customer Value:** Automates invoice ingestion to unlock AP automation and downstream reconciliation.
+
+**REST API Flow:**
+
+1. Start invoice analysis with a URL payload
+2. Poll operation status until completed
+3. Retrieve structured invoice fields
+
+```http
+POST /analyzers/prebuilt-invoice:analyze
+	Content-Type: application/json
+	{
+		"url": "https://.../invoice.pdf"
+	}
+
+GET /analyzers/prebuilt-invoice/operations/{operationId}
+GET /analyzerResults/{operationId}
+```
+
+---
+
+## Create or replace a custom analyzer
+
+**Customer Value:** Tailors extraction to business-specific documents, fields, and structures.
+
+**REST API Flow:**
+
+1. Create or replace analyzer with schema and configuration
+2. Verify analyzer exists
+
+```http
+PUT /analyzers/{analyzerId}
+	Content-Type: application/json
+	{
+		"config": {
+			"schema": { /* field definitions */ },
+			"analysisMode": "...",
+			"processingLocation": "..."
+		}
+	}
+
+GET /analyzers/{analyzerId}
+```
+
+---
+
+## Update, get, list, and delete analyzers
```

Comment:
As far as hero-scenarios go, while this is important, it is also just CRUD operations. Unless there is something unique, I would keep them out.